### PR TITLE
Added ORIGINAL CONTENT skeleton toys that chatter when you play with them

### PR DESCRIPTION
**Pull request by coiax**
I've done a ton of work today. A skele-ton.

:cl: coiax
rscadd: Toy fans, rejoyce! The famous skeleton brothers from the beloved game Belowyarn now have toys that you can win from the arcade machines!
/ :cl:

### CODE REVIEW

**Comment by Cheridan** [comment]
understale is for losers

**Comment by MrPerson** [comment]
4 years from now, will people still recognize this as a reference? Does it sound good to people who don't get the reference?

**Comment by coiax** [comment]
I don't think it sounds awful. And people will probably recognise it as a reference, the sounds are pretty distinctive.

**Comment by Cheridan** [comment]
Sorry but this is in violation of our reference policy.

**Comment by coiax** [comment]
@RemieRichards DEFEND MY HONOUR

**Comment by coiax** [comment]
> reference policy

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -419,16 +419,116 @@ var/regex/lizard_hiSS = new("S+", "g")
...
+
+	var/list/punctuation = list(",", ";", ".", ":", "!", "?")
+	var/list/letter_count = list()
+	for(var/i = 1, i <= words.len, i++)
+		var/word = words[i]
+		// a comma seperated list of punctuation makes my eyes bleed
+		if (word in punctuation)
+			letter_count += word
+			continue
+		letter_count += length(words[i])
```
Comment (duncathan) [inline code/modules/mob/living/carbon/human/species_types.dm]: replace everything below that first comment until here with this
```
var/global/regex/R
if(!R)
	R = regex("(\[\\l\\d]*)(\[^\\l\\d\\s])?", "g") //this regex does that for us!
var/list/L = list()
while(R.next <= length(message))
	R.Find(message)
	if(R.group[1])
		L += length(R.group[1])
	if(R.group[2])
		L += R.group[2]
```

**Comment by PKPenguin321** [comment]
this made me depressed
there was probably a lot of work put into this only for it to get shut down immediately
rip :disappointed:

**Comment by coiax** [comment]
> this made me depressed
> there was probably a lot of work put into this only for it to get shut down immediately
> rip :disappointed:

Certainly sums up my feelings on the matter.

**Comment by Cheridan** [comment]
Seeing as how this is just building onto an existing lame reference I guess I'll let it slide

**Comment by Bawhoppen** [comment]
:-1: does not fit in the game AND is a reference to an even more shit game

**Comment by CosmicScientist** [comment]
@PKPenguin321 could you put together the sounds on puush for the lazy people (me) to hear?

I'm automatically against this due to cringing at the sans and papyrus font reference maymay because I am anti-fun.

**Comment by coiax** [comment]
@CosmicScientist added a link to the demo video in first post

**Comment by xxalpha** [comment]
This sounds awful. The memememe was fine but this is just awful.

**Comment by TheWiznard** [comment]
if you can't add them as actual voices add them as arcade prizes

**Comment by GunHog** [comment]
+1 to the arcade prize idea. It is easier to deal with it, and I can bash someone's face for using it to pester people.

**Comment by CosmicScientist** [comment]
I am anti-fun and therefore my opinion is obvious.

**Comment by coiax** [comment]
Huh, hmm. Well, making some toy skeletons would be fun, and I'd be able to keep all the work I made.

**Comment by Saegrimr** [comment]
Reference or not, its terrible sounding and jarring compared to everything else in the game.

At least the memememe speak was softer.

**Comment by Shadowlight213** [comment]
It sounds terrible

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -413,11 +413,10 @@
 	specflags = list(NOBREATH,HEATRES,COLDRES,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE)
 	var/list/myspan = null
 
-
 /datum/species/skeleton/New()
 	..()
-	myspan = list(pick(SPAN_SANS,SPAN_PAPYRUS)) //pick a span and stick with it for the round
-
+	var/span = pick(SPAN_SANS,SPAN_PAPYRUS) //pick a span and stick with it for the round
+	myspan = list(span)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species_types.dm]: Why? My original form is better.

**Comment by coiax** [comment]
Well, I'm gonna call this as finished as it's gonna be.

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -681,43 +682,129 @@
...
+	while(R.next <= length(message))
+		R.Find(message)
+		if(R.group[1])
+			letter_count += length(R.group[1])
+		if(R.group[2])
+			letter_count += R.group[2]
+
+	spawn(0)
+		// for each word of length N
+		for(var/i = 1, i <=letter_count.len, i++)
```
Comment (RemieRichards) [inline code/game/objects/items/toys.dm]: ```for(var/i in 1 to letter_count.len)```

**Comment by ChangelingRain** [comment]
That's the disposal bin. Feel free to visit it any time.

**Pull request merged**